pkg/rpm_pfg.bzl

```diff
@@ -129,6 +142,72 @@ def _make_absolute_if_not_already_or_is_macro(path):
     # this can be inlined easily.
     return path if path.startswith(("/", "%")) else "/" + path
 
+#### Input processing helper functons.
+
+# TODO(nacl, #459): These are redundant with functions and structures in
+# pkg/private/pkg_files.bzl.  We should really use the infrastructure provided
+# there, but as of writing, it's not quite ready.
```

Comment:
What's the delta?  Is it worth changing there first before this?

Reply:
The main reason for doing this now is I am currently in need this internally to ease migration to the latest and greatest rules_pkg.  I was able to whip this up and fully test it in 90 minutes, and to fully do #459 (which would supplant this change altogether) would take significantly longer.

It is mostly blocked by #385, but even though I could perhaps start without, we would have to rework and/or rewrite much of `pkg_rpm` and `make_rpm.py`.  This change gives us closer API compatibility and reduces user surprise until we can find time to implement the desired backend changes.

Reply:
SGTM

Reply:
Thanks!